### PR TITLE
Add PPP

### PR DESCRIPTION
**Pull request by marcustw**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ You can reach us at the email `e0406158@u.nus.edu`
 <img src="images/marcustw.png" width="200px">
 
 [[github](http://github.com/marcustw)]
-[[portfolio](team/johndoe.md)]
+[[portfolio](team/marcustan.md)]
```
Comment (YuunoKun) [inline docs/AboutUs.md]: Should this be in the form of team/githbub_username_in_lower_case.md? i.e. marcustw

**Review by YuunoKun — APPROVED** [review]
LGTM

**Comment by CodingCookieRookie** [comment]
LGTM

**Review by CodingCookieRookie — APPROVED** [review]
LGTM

File: docs/AboutUs.md (hunk 1 above)
Comment (marcustw) [reply docs/AboutUs.md]: Thank you for pointing it out! I've made the necessary changes.

**Pull request merged**